### PR TITLE
Add support for registering custom drivers

### PR DESCRIPTION
**Pull request by afurculita**
This PR adds support for custom drivers.

I tried to follow as closed as possible the current way of registering drivers.

### CODE REVIEW

**Comment by afurculita** [comment]
ping @henrikbjorn

File: DependencyInjection/BernardExtension.php [hunk 1]
```diff
@@ -15,40 +15,50 @@ protected function loadInternal(array $config, ContainerBuilder $container)
...
                 $this->registerIronMQConfiguration($config['options'], $container);
                 break;
 
             case 'sqs':
+                $container->setAlias('bernard.driver', 'bernard.driver.'.$config['driver']);
                 $this->registerSqsConfiguration($config['options'], $container);
                 break;
 
             case 'pheanstalk':
+                $container->setAlias('bernard.driver', 'bernard.driver.'.$config['driver']);
```
Comment (henrikbjorn) [inline DependencyInjection/BernardExtension.php]: it is a bit excessive to have this in all of the cases. Can we centralize it after the switch?

Comment (afurculita) [inline DependencyInjection/BernardExtension.php]: Done.

**Comment by henrikbjorn** [comment]
Thanks 👍

**Pull request merged**